configure.cmd

```diff
@@ -193,6 +229,21 @@ if "%REACTOS_SOURCE_DIR%" == "%CD%\" (
     cd %REACTOS_OUTPUT_PATH%
 )
 
+if "%VS_SOLUTION%" == "1" (
+
+    if exist build.ninja (
+        echo. && echo Error: This directory has already been configured for ninja.
+        echo An output folder configured for ninja can't be reconfigured for VSSolution.
+        echo Use an empty folder or delete the contents of this folder, then try again.
```

Comment:
We can force this.

Reply:
No, people can have local modifications, we should not delete their shit.

Reply:
Sorry, what I really meant is a flag to do a force cleanup.